src/shared/components/FluxMonacoEditor.tsx

```diff
@@ -17,7 +17,7 @@ import {isFlagEnabled} from 'src/shared/utils/featureFlag'
 // Types
 import {OnChangeScript} from 'src/types/flux'
 import {EditorType} from 'src/types'
-import {editor as monacoEditor} from 'monaco-editor/esm/vs/editor/editor.api.js'
+import {editor as monacoEditor} from 'monaco-editor'
```

Comment:
This was the fix for the right click menu??? wat

Reply:
right?!

Reply:
Holy crud i spent some time looking into this and never saw this. TIL.